src/subaction_animations.jl

```diff
@@ -257,3 +257,34 @@ function _scale(video, action, subaction, rel_frame)
     p = get_interpolation(subaction, rel_frame)
     scale(p)
 end
+
+"""
+    sethue()
+
+Set the color of a function defined inside a [`SubAction`](@ref) using an Animation defined
+with Animations.jl. 
+
```

Comment:
Hey @Wikunia , I am really have difficulty with knowing what I need to do here. I looked at `rotate` and this docstring, but I am still unclear on how to use `sethue()`. I am not sure if it would be useful to put a larger example in the `# Example` section.

Regardless, this functionality definitely needs a tutorial. Maybe we could make tutorials on the Perlin noise and the "please wait" animation you made.

**UPDATE:** I think you forgot to update the `rotate` function based on the test you came up with as `sethue()` is not used there.

Reply:
Yes this needs a tutorial. I think we should make a very short example for all of them such that they show up inside the help mode of the REPL and refer to a tutorial for a bigger picture.

Reply:
I've added one extra line to hopefully make it clearer and the rest should be part of a tutorial.

Reply:
Sounds great, this makes sense to me.